packages/scripts/test/application.unit.ts

```diff
@@ -55,311 +41,6 @@ describe('Application', function () {
         });
     });
 
-    describe('start', () => {
```

Comment:
All of these tests were moved to dev-server.unit.ts, whoever reviews this, this is really important to make sure I didn't change any of the tests here, or missed any, please double check me